subworkflows/local/profiling.nf

```diff
@@ -22,8 +21,14 @@ workflow PROFILING {
     */
 
     // e.g. output [DUMP: reads_plus_db] [['id':'2612', 'run_accession':'combined', 'instrument_platform':'ILLUMINA', 'single_end':1], <reads_path>/2612.merged.fastq.gz, ['tool':'malt', 'db_name':'mal95', 'db_params':'"-id 90"'], <db_path>/malt90]
-    ch_input_for_profiling = shortreads
-            .mix( longreads )
+    ch_input_for_profiling = reads
+            .map {
+                meta, reads ->
+                    def meta_new = meta.clone()
+                        pairtype = meta_new['single_end'] ? '_se' : '_pe'
+                        meta_new['id'] =  meta_new['id'] + pairtype
```

Comment:
It seems I forgot the discussion on this. Why change the ID here again?

Reply:
I also forget, but basically we can't merge two runs, one with PE and one with SE data, together.  So it would result in a crash. 

Initially I said that maybe we can just re-add run accession, however if you've got TWO PE runs and TWO SE runs, we have to remove the run accession information.

At this point the two run-types will get a file name collission at profiling, as we only supply `${meta.id}-${meta.db_name}.report`. Adding `_se`  and `_pe` prevents this